### PR TITLE
Fix  #1092 Update Python/Django versions in tests

### LINKED ISSUE #1092: Update supported Python and Django versions
- drop unsupported versions (e.g. Python 3.5 or Django 3.0)
- add latest versions (e.g. Python 3.10 or Django 4.0)

### PR DESCRIPTION
**Pull request by pauloxnet**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -30,7 +28,7 @@ classifiers =
 
 [options]
 zip_safe=False
-python_requires = >=3.5
+python_requires = >=3.6
 install_requires =
     Django >= 2.2
```
Comment (jschneier) [inline setup.cfg:33]: We should drop 3.0 here

Comment (pauloxnet) [reply setup.cfg:32]: I updated the PR

**Comment by browniebroke** [comment]
Is there anything left to do? How can we help to get this merged?

**Comment by jschneier** [comment]
There are a couple things I think we should update before merging.

1. `install_requires` does not properly drop Django versions
2. CHANGELOG updates should not include `1.12.4`, I wouldn't release a Django or Python dropping version as a patch (did that once and was called out for it, rightly)

**Comment by pauloxnet** [comment]
> There are a couple things I think we should update before merging.

Thanks for the feedback, I updated this PR.

> 1. `install_requires` does not properly drop Django versions

I updated the Django versions in install_requires

> 2. CHANGELOG updates should not include `1.12.4`, I wouldn't release a Django or Python dropping version as a patch (did that once and was called out for it, rightly)

I updated to version 1.13

I also rebased the branch to let you merge it.

**Comment by jschneier** [comment]
Sorry, one more thing. Can you please revert the change to `__version__` and setting the version in `CHANELOG.rst`. For the second you can just put `XXXX-XX-XX`.

**Comment by pauloxnet** [comment]
> Can you please revert the change to `__version__` and setting the version in `CHANELOG.rst`. For the second you can just put `XXXX-XX-XX`.

I reverted the changes on the version, I rebased to the branch master to resolve a merge conflict and I also added the support to Django 4.1 which was published yesterday.

**Comment by jschneier** [comment]
@pauloxnet thanks for the persistence. There is now a test failure.

**Comment by pauloxnet** [comment]
> @pauloxnet thanks for the persistence. There is now a test failure.

@jschneier Actually I had forgotten how old this PR is. I realized that in the meantime Django 2.2 is no longer supported, so I updated this PR. Locally the tests pass, I hope they pass even remotely. Unfortunately, as first-time Contributor for this project they will not automatically start, so I will have to wait for your approval for be sure.
Anyway thanks for your patience.

**Comment by jschneier** [comment]
Merged, thanks for the patience.

**Comment by pauloxnet** [comment]
Thanks to you
I am happy to have been able to contribute

**Pull request merged**